src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int COUNT_OF_FIGURE = 5;
+    private static final int MAX_VALUE_FOR_CALCULATIONS = 10;
+    private static final int DEFAULT_RADIUS_FOR_CIRCLE = 10;
+    private static final int SETTING_FOR_GENERATING_DOUBLE_NUMBER = 3;
+    private static final int SETTING_FOR_GENERATING_RANDOM_NUMBER_MINUS_ONE = 1;
```

Comment:
It may be inlined. See next comments for the explanation. Try to make it simpler. [KISS](https://www.interaction-design.org/literature/topics/keep-it-simple-stupid#:~:text=198%20shares-,What%20is%20Keep%20It%20Simple%2C%20Stupid%20(KISS)%3F,of%20user%20acceptance%20and%20interaction.)